models/stg_netsuite__accounts.sql

```diff
@@ -43,8 +42,7 @@ final as (
         accountnumber as account_number,
         general_rate_type,
         is_leftside,
-        is_balancesheet,
-        _fivetran_deleted
```

Comment:
how come we're deleting _fivetran_deleted everywhere?

Reply:
I am pretty sure the intention was to only remove the where clause that removed the `where not _fivetran_deleted` which makes sense since we still want to report on accounts that have been deleted. However, it would still be beneficial to include the field within the staging model.

Reply:
ah makes sense -- yeah that may be valuable to to pass through